dev/source/docs/ekf2-estimation-system.rst

```diff
@@ -91,8 +91,7 @@ Using EKF2
 
    -  To use only IMU1, set EK2_IMU_MASK = 1 (this is the default)
    -  To use only IMU2, set EK2_IMU_MASK = 2
-   -  To run dual EKF2 using IMU1 and IMU2,  set EK2_IMU_MASK = 3 and
-      turn off the legacy EKF by setting EKF_ENABLE = 0
+   -  To run dual EKF2 using IMU1 and IMU2,  set EK2_IMU_MASK = 3
```

Comment:
```suggestion
   -  To run dual EKF2 using IMU1 and IMU2,  set :ref:`EK2_IMU_MASK<EK2_IMU_MASK> = 3
```
I try to link params if possible so as to create a build error if they get deprecatted to inform me to remove them